raiden-ts/src/raiden.ts

```diff
@@ -1388,6 +1388,36 @@ export class Raiden {
     return receipt.transactionHash as Hash;
   }
 
+  /**
+   * Fetches our current UDC withdraw plan
+   *
+   * @returns Promise to object containing maximum 'amount' planned for withdraw and 'block' at
+   *    which withdraw will become available, and 'ready' after it can be withdrawn with
+   *    [[udcWithdraw]]; resolves to undefined if there's no current plan
+   */
+  public async getUdcWithdrawPlan(): Promise<
+    { amount: UInt<32>; block: number; ready: boolean } | undefined
+  > {
+    const plan = await this.deps.userDepositContract.withdraw_plans(this.address);
+    if (plan.withdraw_block.isZero()) return;
+    return {
+      amount: plan.amount as UInt<32>,
+      block: plan.withdraw_block.toNumber(),
+      ready: plan.withdraw_block.lte(this.state.blockNumber),
+    };
+  }
+
+  /**
+   * Records a UDC withdraw plan for our UDC deposit
+   *
+   * The plan will be ready for withdraw after 100 blocks.
```

Comment:
I would remove the exact number here. Depending on which deployment we use this could theoretically differ. It would be better to keep the SDK independent. For example in future we could have test network deployments with a way shorter plan period time than on production. Or just imagine e2e test deployments.

Reply:
This value is not configurable by deployment options, it's hardcoded, it is exactly 100 blocks, and must be, or else it'd not be this contract, but a different one. It's not a parameter, it's a constant.

Reply:
We discussed that. Keep it if you like it.

Reply:
Removed from the docstring, moved the constant to `constants.ts` :heavy_check_mark: